pkg/orchestrator/provider/kubernetes/container.go

```diff
@@ -72,11 +79,20 @@ func (p *Provider) discoverContainersFromDiscoverer(ctx context.Context, outputC
 		return fmt.Errorf("unable to decode response from discoverer: %w", err)
 	}
 
+	logger := log.GetLoggerFromContextOrDiscard(ctx)
 	for _, container := range containerResponse.Containers {
+		container := container
```

Comment:
not required?

Reply:
Yup. I am aware that this is not required from Go 1.22, but wanted to make sure that the code is backward compatible (can be built with older Go versions where the toolchain does not enforce the Go version defined in `go.mod`) and also make sure that the code base is aligned. We can remove these at some point though.